packages/@webex/plugin-meetings/test/unit/spec/breakouts/index.ts

```diff
@@ -289,5 +292,57 @@ describe('plugin-meetings', () => {
         assert.equal(breakouts.isInMainSession, true);
       });
     });
+
+    describe('#breakoutServiceUrlUpdate', () => {
+      it('sets the breakoutService url', () => {
+        breakouts.breakoutServiceUrlUpdate('newBreakoutServiceUrl');
+        assert.equal(breakouts.breakoutServiceUrl, 'newBreakoutServiceUrl/breakout/');
+      });
+    });
+
+    describe('#toggleBreakout', () => {
+      it('makes the toggleBreakout as expected', async() => {
```

Comment:
might be worth adding a test for the case when `enableBreakoutSession` is already set and verify that `checkLocusDTO()` is not called then, but `doToggleBreakout()` is